packages/tools/client-debugger/client-debugger-view/src/components/ContainerSummaryView.tsx

```diff
@@ -136,40 +175,91 @@ export function ContainerSummaryView(props: ContainerSummaryViewProps): React.Re
 
 	return (
-		<Stack className="container-summary-view">
-			<StackItem>
-				<span>
-					<b>Container</b>:{" "}
-					{containerState.nickname !== undefined
-						? `${containerState.nickname} (${containerState.id})`
-						: containerState.id}
-				</span>
-			</StackItem>
+		<Stack>
 			<StackItem>
-				<span>
-					<b>Status</b>: {statusString}
-				</span>
-			</StackItem>
-			<StackItem>
-				{containerState.clientId === undefined ? (
-					<></>
-				) : (
-					<span>
-						<b>Client ID</b>: {containerState.clientId}
-					</span>
-				)}
-			</StackItem>
-			<StackItem>
-				{containerState.audienceId === undefined ? (
-					<></>
-				) : (
-					<span>
-						<b>Audience ID</b>: {containerState.audienceId}
-					</span>
-				)}
+				<Table size="extra-small" ref={tableRef}>
+					<TableBody>
+						<TableRow>
+							<TableCell
+								{...columnSizing_unstable.getTableCellProps("containerProperty")}
+							>
+								<TableCellLayout>
+									<b>Container</b>
+								</TableCellLayout>
+							</TableCell>
+							<TableCell>
+								<TableCellLayout>{containerState.id}</TableCellLayout>
+							</TableCell>
+						</TableRow>
+						<TableRow>
+							<TableCell>
+								<TableCellLayout>
+									<b>Status</b>
+								</TableCellLayout>
+							</TableCell>
+							<TableCell>
+								<TableCellLayout
+									media={((): JSX.Element => {
+										switch (statusComponents[0]) {
+											case "attaching":
+												return (
+													<Badge shape="rounded" color="warning">
+														{statusComponents[0]}
+													</Badge>
+												);
+											case "detached":
+												return (
+													<Badge shape="rounded" color="danger">
+														{statusComponents[0]}
+													</Badge>
+												);
+											default:
+												return (
+													<Badge shape="rounded" color="success">
+														{statusComponents[0]}
+													</Badge>
+												);
+										}
+									})()}
+								>
+									{statusComponents[1] === "Connected" ? (
+										<Badge shape="rounded" color="success">
+											{statusComponents[1]}
+										</Badge>
+									) : (
+										<Badge shape="rounded" color="danger">
+											{statusComponents[1]}
+										</Badge>
+									)}
+								</TableCellLayout>
+							</TableCell>
+						</TableRow>
+						<TableRow>
+							<TableCell>
+								<TableCellLayout>
+									<b>Client ID</b>
+								</TableCellLayout>
+							</TableCell>
+							<TableCell>
+								<TableCellLayout>{containerState.clientId}</TableCellLayout>
+							</TableCell>
+						</TableRow>
+
+						<TableRow>
+							<TableCell>
+								<TableCellLayout>
+									<b>Audience ID</b>
+								</TableCellLayout>
+							</TableCell>
+							<TableCell>
+								<TableCellLayout>{containerState.audienceId}</TableCellLayout>
+							</TableCell>
+						</TableRow>
+					</TableBody>
+				</Table>
```

Comment:
I think we should consider enforcing a smaller tab space size. Things get pretty nested with html and it becomes hard to read

Reply:
You can set up the "visual size" of a tab as a personal preference [in Github](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/managing-your-tab-size-rendering-preference) and in VSCode:

![image](https://user-images.githubusercontent.com/716334/230427593-c3c15732-1b77-48d4-8a9e-b891e1db5847.png)